ui/lib/utils.ts

```diff
@@ -16,3 +17,11 @@ export const getNavValue = (currentPage: any): PageRoute | boolean => {
       return false;
   }
 };
+
+export function notifySuccess(message: string) {
+  toast["success"](message);
+}
+
+export function notifyError(message: string) {
+  toast["error"](`Error: ${message}`);
```

Comment:
🔥 🍞 .

I think we are using Snack not Toast cc @AlinaGoaga

Reply:
Not married to the lib. It should be easy enough to swap out.